app/policies/audio_file_policy.rb

```diff
@@ -0,0 +1,5 @@
+class AudioFilePolicy < AccountablePolicy
+  def original?
+    update?
```

Comment:
I don't have your changes yet for using tokens in this PR, should I merge them in, or does this work even without the changes in #115 ?